### PR TITLE
Wave2 558 (un)publish enhancement

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc [hunk 1]
```diff
@@ -57,11 +57,55 @@ function _fpp_publish() {
...
+                        }
+                    }
+
+                }
+
                 // Now, actually send the mails.
                 $params = array(
                     'subject' => 'content published',
-                    'message' => 'your brick titled '. $n->title .' is published',
+                    'message' => 'your brick titled '. $n->title .' is published. This brick can be found on following page '.$GLOBALS['base_url'].'/'.$fpp_page,
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc]: Minor grammar issue: Can be added on "the following".

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc [hunk 2]
```diff
@@ -131,10 +175,54 @@ function _fpp_unpublish() {
...
+                    }
+
+                }
+
+
                 // Now, actually send the mails.
                 $params = array(
                     'subject' => 'content unpublish notification',
-                    'message' => 'your brick titled ' . $n->title . ' is unpublished',
+                    'message' => 'your brick titled '. $n->title .' is un-published. This brick can be found on following page: '.$GLOBALS['base_url'].'/'.$fpp_page,
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc]: Same issue as above.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc [hunk 3]
```diff
@@ -224,10 +312,54 @@ function _fpp_advance_unpub_notification()
...
+                            }
+
+                        }
+
+
                         // Now, actually send the mails.
                         $params = array(
                             'subject' => 'content unpublish notification',
-                            'message' => 'your brick titled ' . $n->title . ' is scheduled to unpublish in' . $advance_notification,
+                            'message' => 'your brick titled '. $n->title .' is scheduled to un-publish in' . $advance_notification .'. This brick can be found on following page: '.$GLOBALS['base_url'].'/'.$fpp_page,
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc]: Same issue as above.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc [hunk 4]
```diff
@@ -57,11 +57,55 @@ function _fpp_publish() {
                 $mail = $record['mail'];
                 $from = 'wem@ucalgary.ca';
 
+                //Now get the page of this fpid, so we can get the link to page containing this FPP for the email
+
+
+                //get the serialized conf data from page manager handler because that is only way I found to access correct display id for panels table
+                $query = db_select('page_manager_handlers', 'pmh');
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_fpp_scheduling/ucgy_core_fpp_scheduling.cron.inc]: Correct me if I'm wrong, but this appears to be the first of three mostly duplications of code. Can this be added as a function and be called in the other two instances where this code is needed?

**Comment by nindusaini** [comment]
Requested changes made and tested.

**Pull request merged**